cmd/clusterctl/client/config/providers_client.go

```diff
@@ -44,6 +44,7 @@ const (
 	DockerProviderName         = "docker"
 	DOProviderName             = "digitalocean"
 	GCPProviderName            = "gcp"
+	HarvesterProviderName      = "harvester"
```

Comment:
It should contain the github org to distinct from potential community projects:

https://main.cluster-api.sigs.k8s.io/developer/providers/contracts/clusterctl.html?highlight=provider#providers-github-org-prefix

Reply:
Does this also need to be changed in the docs ?